generate-matrix/action.yml

```diff
@@ -0,0 +1,28 @@
+name: Generate Matrix
+description: Sub-action to generate a matrix of stack-yamls
+inputs:
+  working-directory:
+    description: "Working directory for run commands"
+    required: true
+    default: .
+  find-options:
+    description: "Arguments to find(1) stack-yaml files"
+    required: true
+    default: "-maxdepth 1 -name 'stack*.yaml'"
```

Comment:
For the record, this can be overridden to append something like `-not -name stack-something.yaml` to accomplish an "exclude"-like option. However, it will probably make sense to add something specifically for that if we find there's a need.